lib/utils/import/multiple/index.ts

```diff
@@ -33,81 +30,45 @@ class MultipleImporter extends EventEmitter {
 
     for (let i = 0; i < filesArray.length; i++) {
       const file = filesArray[i];
-      if (i + 1 === filesArray.length) {
-        lastFileName = file.name;
-      }
       const fileExtension =
         file.name.substring(file.name.lastIndexOf('.') + 1, file.name.length) ||
         file.name;
 
-      if (fileExtension === 'md' || fileExtension === 'txt') {
-        this.textFiles.push(file);
-      } else if (fileExtension === 'json') {
-        this.simplenoteFiles.push(file);
-      } else if (fileExtension === 'enex') {
-        this.evernoteFiles.push(file);
-      }
+      // Get the required importer based on the current files extension
+      // Add the file to this importers list of files
+      const theImporter = importers.find((importer) => {
+        return importer.fileTypes.indexOf(fileExtension) !== -1;
+      });
```

Comment:
another place where having the function in the `importers` module would be nice.

```ts
const importer = importers.forFilename(file);
```

Reply:
I'm noting here that with this one there's an implicit rule that the first importer to support importing the file extension is the one we'll choose. we can declare this rule when we move the logic into `importers.ts` or we can set up an explicit mapping between filename/extension and chosen importer to remove the ambiguity, for example, a switch statement on file extension.

Reply:
This all makes sense and I'll work on it. Thank you!